From stable+bounces-56128-greg=kroah.com@vger.kernel.org Sat Jun 29 17:14:15 2024
From: Ard Biesheuvel <ardb+git@google.com>
Date: Sat, 29 Jun 2024 17:13:58 +0200
Subject: drivers: fix typo in firmware/efi/memmap.c
To: stable@vger.kernel.org
Message-ID: <20240629151357.866803-6-ardb+git@google.com>

From: Zheng Zhi Yuan <kevinjone25@g.ncu.edu.tw>

[ Commit 1df4d1724baafa55e9803414ebcdf1ca702bc958 upstream ]

This patch fixes the spelling error in firmware/efi/memmap.c, changing
it to the correct word.

Signed-off-by: Zheng Zhi Yuan <kevinjone25@g.ncu.edu.tw>
Signed-off-by: Ard Biesheuvel <ardb@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/firmware/efi/memmap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/firmware/efi/memmap.c
+++ b/drivers/firmware/efi/memmap.c
@@ -245,7 +245,7 @@ int __init efi_memmap_install(struct efi
  * @range: Address range (start, end) to split around
  *
  * Returns the number of additional EFI memmap entries required to
- * accomodate @range.
+ * accommodate @range.
  */
 int __init efi_memmap_split_count(efi_memory_desc_t *md, struct range *range)
 {
